Remove useless inclusion of X11/Intrinsic.h in common sources so that xawtv
can be built without X at all.

Index: xawtv-3.95/common/midictrl.c
===================================================================
--- xawtv-3.95.orig/common/midictrl.c
+++ xawtv-3.95/common/midictrl.c
@@ -6,7 +6,6 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <pthread.h>
-#include <X11/Intrinsic.h>
 
 #include "grab-ng.h"
 #include "commands.h"
Index: xawtv-3.95/common/vbi-data.c
===================================================================
--- xawtv-3.95.orig/common/vbi-data.c
+++ xawtv-3.95/common/vbi-data.c
@@ -14,7 +14,6 @@
 #include <assert.h>
 #include <pthread.h>
 
-#include <X11/Intrinsic.h>
 
 #include "vbi-data.h"
 #include "vbi-sim.c"
